testsuite/MDAnalysisTests/topology/test_lammpsdata.py

```diff
@@ -179,6 +180,26 @@ def test_traj(self, filename):
                               dtype=np.float32))
 
 
+class TestLammpsDataPairIJ(LammpsBase):
+    """Tests the reading of lammps .data topology file with a
+    PairIJ Coeffs section
+    """
+    
```

Comment:
```suggestion

```
No worries about the darker linting, as long as it's not breaching PEP8 it ends up just `black` trying to enforce its style.

That being said the above "whitespace on an empty line" is a PEP8 violation

Sorry about this, as @orbeckst keeping maintenance  costs down ends up requiring a lot of weird formatting rules that do tend to be rather unfriendly to new contributors :(